README.md

```diff
@@ -1,2 +1,127 @@
 #Idea Town
-The addon where ideas come to idea
\ No newline at end of file
+The addon where ideas come to idea
+
+## installation
+
+`npm install`
+
+## configuration
+
+check the `CONFIG` property in `package.json`
+
+## running
+
+* `npm start`
+* open http://localhost:1337/stub-content/index.html
```

Comment:
I think you need a `jpm run` between these two steps, to fire up a new FF instance with the addon installed.